__tests__/__src__/environment/TestEnvironment.ts

```diff
@@ -40,7 +40,7 @@ export class TestEnvironment extends BaseTestEnvironment {
 
         // Ensure correct path separator for windows or linux like systems.
         const separator = process.platform === "win32" ? ";" : ":";
-        result.env.PATH = `${nodePath.resolve(__dirname, "../../__resources__/application_instances")}${separator}${process.env.PATH}`;
+        result.env.PATH = `${process.env.PATH}${separator}${nodePath.resolve(__dirname, "../../__resources__/application_instances")}`;
```

Comment:
Would this change cause integration/system tests on our local machines to use a different Zowe CLI, and if so is this desirable?

For example, if `%APPDATA%\npm` is already on my PATH, won't the Zowe bin there take priority over the local CLI instance because of this change?

Reply:
you are right, this change should not be needed for the purpose of running the Integration tests with the binaries : )
reverting